fundamentals/examples.tex

```diff
@@ -151,79 +151,111 @@ \subsection{Simulations}
-reactor simulation, open (no recycle), modified open, and closed (infinite-pass recycle) 
-fuel cycles are compared to illustrate the resolution possible with \Cyclus.  The simulation
-is then expanded to include multiple reactors in the same set of fuel cycles to demonstrate
-the system-scale convergence to continuous material flow models. 
-
-Because material is tracked as discrete objects, single-pass recycle using \gls{MOX} 
-fuel is easy to implement in \Cyclus.  The \gls{MOX} simulation includes one provider of fresh
-\gls{UOX} fuel, one recycled fuel fabrication facility, one reactor, one separations
-facility, and one repository. The simulation has a duration of 1100 months and a \TODO{??-month stepsize}.
-
-The \Class{BatchReactor} archetype from \Cycamore has the ability to accept
-fuel materials from an arbitrary number of different sources, delineated by
-commodity. Custom, non-uniform preferences can be set for each different fuel
-commodity.  The \gls{DRE} enables a reactor to
-preferentially accept recycled \gls{MOX} fuel over fresh \gls{UOX} fuel.  In this
-simulation, the
-\Class{BatchReactor} was configured to run with a 3 batch core on 18 month cycles and
-a 2-month refueling period.  Adjusting and setting parameters such as these is
-as simple as editing numbers in an input file such as:
-
-\begin{lstlisting}[language=xml]
-  ...
-  <BatchReactor>
-      ...
-      <processtime>18</processtime>
-      <nbatches>3</nbatches>
-      <batchsize>20000</batchsize>
-      <refueltime>2</refueltime>
-      ...
-  </BatchReactor>
-  ...
-\end{lstlisting}
+This section will discuss the creation and results of a few simple fuel cycle
+simulations to help illustrate the flexibility of \Cyclus. These simulations
+are designed to illustrate Cyclus' capability.  Many simplifying assumptions
+have been made with respect to material compositions, fuel transmutation, and
+others. The three fuel cycles examined are:
+
+\begin{enumerate}
+    \item Once through (no recycle) 
+    \item Modified open (1-pass recycle)
+    \item closed (infinite-pass recycle)
+\end{enumerate}
+
+For each of these fuel cycles, a 1,100 month \Cyclus simulation with only a
+single reactor is run in addition to a simulation with 10 reactors with
+staggered refueling times.  As the number of staggered-cycle reactors
+increases the system converges toward continuous material flow results.
+\Cyclus flexibility allows this transition to be examined.  The facilities
+used in these simulations are:
+
+\begin{itemize}
+
+    \item \textbf{Reactor} (\Class{cycamore::Reactor}): This is a reactor
+        model that requests any number of input fuel types and uses static
+        compositions to transmute them to after they are burnt and discharged
+        from the core.  Due to the static spent fuel compositions spent fuel
+        compositions become "incorrect" as the input fuel diverges from the
```

Comment:
I would suggest just dropping this sentence about the compositions becoming incorrect. That subtlety is beyond the scope of this itemized list, which is intended to give enough information to help someone reproduce the simulation.